functions/_fk_script.fish

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env fish
+
```

Comment:
what is this and why is it needed?

Reply:
This is a helper function for calling ruby scripts. This isn't needed right now but will be useful later.